packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -95,36 +91,39 @@ function LoadingCard(props: { numPlots: number }) {
     </Paper>
   )
 }
-const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
-  title,
-  description,
-  facetsToPlot,
-  detailsPagePath,
-}: FacetPlotsCardProps): JSX.Element => {
+
+function FacetPlotsCard(props: FacetPlotsCardProps) {
+  const { title, description, facetsToPlot, detailsPagePath } = props
   const { accessToken } = useSynapseContext()
-  const isLoadingNewBundle = useAtomValue(isLoadingNewBundleAtom)
-  const data = useAtomValue(tableQueryDataAtom)
+  const { queryMetadataQueryOptions } = useQueryContext()
+  const { data: queryMetadata } = useSuspenseQuery(queryMetadataQueryOptions)
```

Comment:
Use `useSuspenseQuery` in a few places in this component so it suspends until it's gathered all of the data needed to show the plot.